README.md

```diff
@@ -1298,6 +1299,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Rutik Bhoyar](https://github.com/Rutikab12)
   - [Rwitesh Bera](https://github.com/rwiteshbera)
   - [Ryga](https://github.com/ryga9)
+
```

Comment:
Please remove this extra line.
```suggestion
```

Reply:
Its showing error. Should i still go with the commit??
![image](https://user-images.githubusercontent.com/8182816/195173936-ad9efc5f-7c85-4527-959e-a0827d3c5a53.png)